frontend/src/LandingPage.js

```diff
@@ -6,19 +6,18 @@ import { Link } from 'react-router-dom';
 import { createBrowserHistory } from "history";
 import SECRET_KEY from './config';
 import ITINERARY_KEY from './configMap';
-import Map from './Itinerary';
 import Button from "react-bootstrap/Button";
 import ReactDOM from 'react-dom';
 import ImgKey from './configImg';
 import DayPickerInput from 'react-day-picker/DayPickerInput';
 import 'react-day-picker/lib/style.css';
 
-let autoComplete1, autoComplete2, currentUserId;
-let lat1, lat2, long1, long2, url;
+let autoComplete1, autoComplete2, autoComplete3, currentUserId;
+let lat1, lat2, long1, long2, lat3, long3, url;
```

Comment:
The names that only differ in one letter, discouraged. It's easy to make a typo here. Consider renaming it to something more describing.